src/newSeed/seedConfig.ts

```diff
@@ -61,4 +68,7 @@ export class SeedConfig implements ISeedConfig {
   public tokenDetails = {
     tokenDistrib: [],
   } as ITokenDetails;
+  public contactDetails={
+    remarks: "",
```

Comment:
I'm just curious why this one is initialized and the other two aren't?

Reply:
Only this input field needs to be initialized, the others email and project logo are to be filled out. The initialization is to set the length to 0 initially so that i can display that on the ui